### DISCUSSION TITLE
request help: How to save the jwt-auth plugin  parsed content

### DISCUSSION THREAD

**Discussion by illmy** [apache/apisix#5017] [Q&A]
### Issue description

I want to get the jwt-auth plugin parsed content

### Environment

- apisix version (cmd: `apisix version`): 2.8
- OS (cmd: `uname -a`): Linux
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
- apisix-dashboard version, if have: 2.7
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

**Comment by tzssangglass** [comment]
see: https://github.com/apache/apisix/blob/717760c2dfb9dcd31b2072cf805a852e200798d8/apisix/plugins/jwt-auth.lua#L262-L263

and: https://github.com/SkyLothar/lua-resty-jwt#load--verify
